sagemaker-fundamentals/create-training-job/create_training_job_vpc.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
<p>Unfortunately the LocationConstraint fails when the region is <code>us-east-1</code> since that is the default region. Boto3 S3 docs note this. So I think you need to add logic to only use the LocationConstraint if region != us-east-1.</p><p> </p><p>Also typo in comment, <code>exisit</code>  instead of <code>exist</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Full error message:</p><pre class="ql-syntax" spellcheck="false">ClientError: An error occurred (InvalidLocationConstraint) when calling the CreateBucket operation: The specified location-constraint is not valid
</pre><p> </p>

Reply:
<p>added location check</p>